docs/gettinginvolved/virtual-machines.rst

```diff
@@ -0,0 +1,52 @@
+.. _building-flocker-virtual-machine-images:
+
+Building Flocker Virtual Machine Images
+=======================================
+
+Flocker virtual machine images can be built using a tool called Packer.
+The Flocker source repository has an ``admin/packer`` sub-directory which contains Packer templates and provisioning scripts.
+These are used to create Ubuntu AMI images for use in the Flocker Cloudformation demonstration environment.
```

Comment:
At a guess, the sphinx tests are failing because "Cloudformation" looks like a spelling mistake. Please fix them.